scipy/linalg/_sketches.py

```diff
@@ -26,12 +26,12 @@ def cwt_matrix(n_rows, n_columns, seed=None):
         Number of rows of S
     n_columns: int
         Number of columns of S
-    seed : None or int or `numpy.random.RandomState` instance, optional
-        This parameter defines the ``RandomState`` object to use for drawing
-        random variates.
-        If None (or ``np.random``), the global ``np.random`` state is used.
-        If integer, it is used to seed the local ``RandomState`` instance.
-        Default is None.
+    seed : {None, int, `numpy.random.Generator`}, optional
+        If `seed` is None the `numpy.random.Generator` singleton is used.
```

Comment:
There is no `Generator` singleton. The default behavior of the `check_random_state()` function that this method uses will still use the `RandomState` singleton when `seed is None`. Similarly, when `seed` is an integer, you will get a `RandomState` instance, not a `Generator`.

The old version in the `rvs()` method down below is the most accurate description. Basically, old versions that don't mention `Generator` should be updated to include that option, but these new versions that omit references to `RandomState` entirely are currently incorrect.